### PR TITLE
82 dry up views closes #82

### LINKED ISSUE #82: Dry up headers into a partial
we are repeating ourselves with the header and spacer too much already

need to put it into a partial

### LINKED ISSUE #97: Job seeds don't have tags
Jobs generated with seeds file do not tags associated with them.

### PR DESCRIPTION
**Pull request by HoffsMH**
This pull request pulls common elements on our site into partials such as the header and navbar links

in addition it updates the javascripts to let us filter correctly

I also went through each view and made sure that we are viewing pages correctly when both logged in and out

closes #97

### CODE REVIEW

File: app/views/businesses/new.html.erb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <% content_for :navbar_links do %>
-<li><%= link_to "Edit Profile", logout_path%></li>
+  <%= render 'layouts/basic_navbar/user_links' %>
 <% end %>
 
-<div class="page-heading">
-  <div class="container">  <h1>New Business</h1></div>
-</div>
+<%= render "layouts/page_heading", locals = {heading: "#{current_user.nickname}: New Business"}%>
```
Comment (jphoenix86) [inline app/views/businesses/new.html.erb]: heading: "jphoenix86: New Business"

What does this mean?

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -1,6 +1,12 @@
+<% content_for :navbar_links do %>
+  <%= render 'layouts/basic_navbar/user_links' %>
+<% end %>
+
+<%= render "layouts/page_heading", locals = {heading: "Edit Profile: #{current_user.nickname"}%>
```
Comment (7hoenix) [inline app/views/users/edit.html.erb:5]: This is backwards the link above (new business path)

above its => "mhecker: New Business"
Here its => "Edit Profile: mhecker"

Let's make those match.

Comment (HoffsMH) [inline app/views/users/edit.html.erb:5]: My thinking was that 

mhecker is making a new business
you are editing the profile of mhecker

I'll change it to be Title: username in both cases

for consistency

**Comment by HoffsMH** [comment]
Are there any other instances where the ordering on the header is in consistent?

**Comment by HoffsMH** [comment]
?

**Comment by HoffsMH** [comment]
Another issue here is that, In order to display a button for each available tag in the database we need all the tags

before I was using javascript_include tag to include something from our assets but with this method I could not pass the list of available tags to the javascript.

As long as we are counting this particular script as an asset it needs to be recompiled each time a new tag is added by an admin which is not feasable.

**Comment by HoffsMH** [comment]
I added that this closes #97  since in the pr jobs also have random tags now

**Pull request merged**